content-src/components/DiscoveryStreamComponents/DSEmptyState/DSEmptyState.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+export class DSEmptyState extends React.PureComponent {
+  render() {
+    return (
+      <div className="section-empty-state">
+        <p className="empty-state-message">
```

Comment:
This isn't a blocker, but as we begin to address accessibility, i think we should use semantic markup whenever possible. With that in mind I recommend we change this to…

```
<div className="section-empty-state">
  <h2>You're caught up!</h2>
  <p>Check back later for more stories.</p>
</div>
```